### PR TITLE
fix(web): aggs data key

### PR DESCRIPTION
**Pull request by ShahAnuj2610**

### CODE REVIEW

File: packages/web/src/components/search/CategorySearch.js [hunk 1]
```diff
@@ -825,13 +824,14 @@ class CategorySearch extends Component {
...
 	getComponent = (downshiftProps = {}) => {
-		const { error, isLoading } = this.props;
+		const { error, isLoading, aggregationData } = this.props;
 		const { currentValue } = this.state;
 		const data = {
 			error,
 			loading: isLoading,
 			downshiftProps,
 			data: this.parsedSuggestions,
+			aggregationData,
```
Comment (bietkul) [inline packages/web/src/components/search/CategorySearch.js:834]: @ShahAnuj2610 Do we return `aggregationData` in `onData` method too?

File: packages/web/src/components/search/DataSearch.js [hunk 2]
```diff
@@ -191,9 +190,9 @@ class DataSearch extends Component {
 	// returns size and aggs property
 	getBasicQueryOptions = () => {
 		const { aggregationField, size } = this.props;
-		let queryOptions = { size };
+		const queryOptions = { size };
 		if (aggregationField) {
-			queryOptions = getCompositeAggsQuery({ ...queryOptions }, this.props, null, true);
+			queryOptions.aggs = getCompositeAggsQuery({}, this.props, null, true).aggs;
```
Comment (bietkul) [inline packages/web/src/components/search/DataSearch.js:195]: @ShahAnuj2610 Why did you change this logic?
You're reading the function value without ensuring that the function returns an object which can break the app.

**Review by bietkul — CHANGES_REQUESTED** [review]

File: packages/web/src/components/search/CategorySearch.js (hunk 1 above)
Comment (ShahAnuj2610) [reply packages/web/src/components/search/CategorySearch.js:834]: yes, but `onData` is only for `ReactiveComponent` and `ReactiveList`

Comment (bietkul) [reply packages/web/src/components/search/CategorySearch.js:834]: I mean `onSuggestions` in that case.

File: packages/web/src/components/search/DataSearch.js (hunk 2 above)
Comment (ShahAnuj2610) [reply packages/web/src/components/search/DataSearch.js:195]: https://github.com/appbaseio/reactivecore/blob/01b710f1862dbb8101919bee5c3d6f143d4c29fb/src/utils/helper.js#L450 => it returns an object

File: packages/web/src/components/search/CategorySearch.js (hunk 1 above)
Comment (bietkul) [reply packages/web/src/components/search/CategorySearch.js:834]: @ShahAnuj2610 I checked it and found that we don't have `onData` in search components, so you can leave it for now.
